### PR TITLE
Add fastify-vuejs to the ecosystem's lists

### PR DESCRIPTION
**Pull request by lolo32**
This is the official Vue.js Webpack template ported to Fastify, with all tests integrated

### CODE REVIEW

File: docs/Ecosystem.md [hunk 1]
```diff
@@ -41,4 +41,5 @@ Plugins maintainted by the fastify team are listed under [Core](#core) while plu
 - [`fastify-server-session`](https://github.com/jsumners/fastify-server-session) A session plugin with support for arbitrary backing caches via `fastify-caching`.
 - [`fastify-session`](https://github.com/SerayaEryn/fastify-session) a session plugin for Fastify.
 - [`fastify-sse`](https://github.com/lolo32/fastify-sse) to provide Server-Sent Events with `reply.sse( … )` to Fastify.
+- [`fastify-vuejs`](https://github.com/lolo32/fastify-vuejs) a Vue.js starting template, from original [Vue.js webpack template](https://github.com/vuejs-templates/webpack), ported to fastify, with all tests activated.
```
Comment (jsumners) [inline docs/Ecosystem.md]: "fastify" => "Fastify"

**Comment by jsumners** [comment]
@delvedor is it necessary for the downstream project to have more comprehensive documentation to be included in the "Community" list?

**Comment by delvedor** [comment]
@jsumners I think that if we list a plugin in our ecosystem documentation, that plugin should be well written and documented.
From the documentation of [`fastify-vuejs`](https://github.com/lolo32/fastify-vuejs) I can't understand how it works and how use it.

**Comment by jsumners** [comment]
Okay, as long as we apply the rule consistently, it's fine with me.

**Comment by delvedor** [comment]
@jsumners we should continue the discussion started in https://github.com/fastify/fastify/issues/294.

**Comment by mcollina** [comment]
Closing for no activity, feel free to reopen if you still want to contribute this!

**Pull request closed without merging**